src/app/@ansyn/imagery/communicator-service/communicator.entity.ts

```diff
@@ -279,6 +279,9 @@ export class CommunicatorEntity implements OnInit, OnDestroy {
 		const { worldView: { mapType, sourceType }, data: { position } } = this.mapSettings;
 		this.setActiveMap(mapType, position, sourceType).then(() => {
 			this.imageryCommunicatorService.createCommunicator(this);
+			const hideLayers = this.mapSettings.flags && this.mapSettings.flags.hideLayers;
```

Comment:
you can also change to this.mapSettings?.flags?.hideLayers
instead of this.mapSettings.flags && this.mapSettings.flags.hideLayers;